ocaml/src/alba_client_upload.ml

```diff
@@ -419,18 +442,63 @@ let upload_object''
            ~object_id ~object_name
            ~chunk:chunk' ~chunk_size:chunk_size_with_padding
            ~chunk_id
-           ~k ~m ~w'
+           ~k ~m ~w' ~min_fragment_count
            ~compression ~encryption ~fragment_checksum_algo
            ~version_id ~gc_epoch
            ~object_info_o
            ~osds:target_osds
            ~fragment_cache
-           ~cache_on_write)
+           ~cache_on_write
+           ~upload_slack
```

Comment:
only add slack for the last chunk?
`~upload_slack:(if has_more then 0.0 else upload_slack`